smf/nexus/manifest.xml

```diff
@@ -4,28 +4,41 @@
 <service_bundle type='manifest' name='nexus'>
 
 <service name='oxide/nexus' type='service' version='1'>
-  <create_default_instance enabled='false' />
+  <create_default_instance enabled='true' />
 
   <dependency name='multi_user' grouping='require_all' restart_on='none'
     type='service'>
   <service_fmri value='svc:/milestone/multi-user:default' />
   </dependency>
+
   <dependency name='network' grouping='require_all' restart_on='none'
     type='service'>
   <service_fmri value='svc:/milestone/network:default' />
+
```

Comment:
nit: inconsistent blank line.

Reply:
gargh, merged without fixing this, I'll get it in the Internal DNS PR